tests/providers/osfstorage/fixtures/root_provider.json

```diff
@@ -1,175 +1,178 @@
 {
-   "folder_metadata":{
```

Comment:
`root_provider.json` has weird indentation with 3 spaces. Use 2 spaces instead. No change to this file except this style one.